x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -29,6 +29,11 @@ class org.elasticsearch.xpack.ql.expression.function.scalar.whitelist.InternalQl
   double nullSafeSortNumeric(Number)
   String nullSafeSortString(Object)
 
+#
```

Comment:
Same here, there is a block for ascii functions: https://github.com/elastic/elasticsearch/pull/56404/files#diff-edef662ace9df6873edfc33572672eeeR144

Reply:
Yep, same here. Different ScriptUtils class that feeds the painless method in sql.